poc/tests/test_field.py

```diff
@@ -49,6 +49,43 @@ def run_ntt_field_test(self, cls: type[NttField]) -> None:
         # Test generator.
         self.assertTrue(cls.gen()**cls.GEN_ORDER == cls(1))
 
+        for logN in range(1, 8):
```

Comment:
Let's follow python naming conventions. Here and below:
```suggestion
        for log_n in range(1, 8):
```